src/Mod/TechDraw/App/LineGenerator.h

```diff
@@ -42,6 +42,8 @@
 #include <map>
 #include <QPen>
 
+#include <map>
```

Comment:
Why does this need a second include?

Reply:
It doesn't.  I have a rebase which will make this PR play nicely with the current `main` branch.  I'll click `Ready for review` when it passes CI.  This should happen later today.